target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4a-100m-intl.dts

```diff
@@ -5,6 +5,28 @@
 / {
 	compatible = "xiaomi,mi-router-4a-100m-intl", "mediatek,mt7628an-soc";
 	model = "Xiaomi Mi Router 4A (100M International Edition)";
+
+	aliases {
+		led-boot = &led_power_yellow;
+		led-failsafe = &led_power_yellow;
+		led-running = &led_power_blue;
+		led-upgrade = &led_power_yellow;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_blue: power_blue {
+			label = "blue:power";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_yellow: power_yellow {
+			label = "yellow:power";
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
```

Comment:
You are adding the led entries here, but you haven't removed them from the `mt7628an_xiaomi_mi-router-4.dtsi` did you miss it there?

Reply:
Since moving the led entries  was the point of the first stage patch, looks like I kind of messed this up in the process.
Coming back with a new patch after I have redone the dtdiff and verified nothing changes.

Reply:
You only deleted the alias in the `target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4.dtsi` file, the leds node was not changed. 
The dtb is not changed because you overwrite the leds with the same definitions in the dts files.